.github/workflows/update_package_properties.yml

```
@@ -0,0 +1,44 @@
+name: Update Package Properties
+
+env:
+  PRJ_NAME: "citus"
+on:
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        description: 'The tag name on the project repo to be released'
+      fancy_version_no:
+        description: 'If fancy versioning is needed, set value greater than 1 e.g 2'
+        default: '1'
+      microsoft_email:
+        description: 'Email to be written on changelogs'
+        default: 'gindibay@microsoft.com'
+      name:
+        description: 'Name to be written on changelogs'
+        default: 'Gurkan Indibay'
+jobs:
+  update_package_properties:
```

```suggestion
  update_package_properties:
    name: Update Package Properties
```

I left a comment https://github.com/citusdata/packaging/pull/645#discussion_r611405107 with 2 parts. You addressed the first item there, and missed the second one

---

Fixed

---

How is this fixed exactly? I do not see any field for `jobs.update_package_properties.name` here.

---

I hadn't realized the job name field until you clarify with the comment above. I didn't get used to the structure of comments half is here half is up somewhere.
By the way, when you solve the comments I can track the comments more easily.

---

Fixed as you suggested